src/wallet/rpcdump.cpp

```diff
@@ -1258,7 +1258,7 @@ static UniValue ProcessImport(CWallet * const pwallet, const UniValue& data, con
 
         // All good, time to import
         pwallet->MarkDirty();
-        if (!pwallet->ImportScripts(import_data.import_scripts)) {
+        if (!pwallet->ImportScripts(import_data.import_scripts, timestamp)) {
```

Comment:
In commit "Optionally allow ImportScripts to set script creation timestamp" (ace5d082034cbaa52f596689433381a2226a21dc)

Can you clairfy in the commit message whether this is a change in behavior for the `importmulti` RPC. Seems like a minor bugfix?

Reply:
Done.